poster/styles/styles.scss

```diff
@@ -11,480 +7,451 @@ sans-serif;
-    margin-left: 4vw;
-}
-
-.body-lineeight {
-    margin-left: 4vw;
-}
-
-.body-lineten {
-    margin-left: 12vw;
-}
-
-.body-lineeleven {
-    margin-left: 8vw;
-}
-
-.body-linetwelve {
-    margin-left: 12vw;
-    margin-bottom: -2vw;
-}
-
-.body-linethirteen {
-    margin-top: -1.5vw;
-    margin-left: 12vw;
-}
-
-.body-linefourteen {
-    margin-left: 8vw;
-}
-
-.body-linefifteen {
-    margin-left: 4vw;
-}
-
-.body-linesixteen {
-    margin-left: 8vw;
-}
-
-.body-lineeighteen {
-    margin-left: 4vw;
-}
-
-.body-linenineteen {
-    margin-left: 6vw;
-}
-
-.body-linetwenty {
-    margin-left: 12vw;
-}
-
-.body-linetwentyone {
-    margin-left: 16vw;
-}
-
-.body-linetwentytwo {
-    margin-top: 10vw;
-    margin-left: 20vw;
-}
-
-.body-linetwentythree {
-    margin-top: 0;
-    margin-left: 16vw;
-}
-
-.body-linetwentyfour {
-    margin-top: -1vw;
-    margin-left: 14vw;
-    margin-bottom: -1.5vw;
-}
-
-.body-linetwentyfive {
-    margin-left: 7vw;
-}
-
-.body-linetwentysix {
-    margin-left: 10vw;
-}
-
-.body-linetwentyseven {
-    margin-left: 8vw;
-}
-
-.body-linetwentyeight {
-    margin-left: 6vw;
-}
-
-.body-linetwentynine {
-    margin-left: 4vw;
-}
-
-.body-linethirty {
-    margin-left: 16vw;
-    margin-bottom: -1.5vw;
-}
-
-.body-linethirtyone {
-    margin-left: 8vw;
+body {
+    background: #202221;
 }
 
-.body-linethirtytwo {
-    margin-left: 4vw;
+/*header in top half*/
+h1 {
+    font-size: 24vw;
+    margin-top: -10vw;
+    margin-bottom: 0;
+    display: grid;
+    grid-template-columns: repeat(7, 14vw);
+    grid-template-rows: repeat(20, 4vw);
 }
 
-.body-linethirtytwosecond {
-    margin-left: 2vw;
+/*container for whole compostion separating in top and bottom halves*/
+.outerwrapper {
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: 1fr 1fr;
+    grid-gap: 0;
+    color: #FAFAF2;
 }
 
-.body-linethirtythree {
-    margin-left: 8vw;
+/*title location within outer wrapper*/
+.tophalf {
+    grid-row: 1/2;
 }
 
-.body-linethirtyfour {
-    display: inline-block;
-    margin-top: 2vw;
-    margin-left: 18vw;
-}
+/*bottom half grid establishing 2 columns and text properties*/
+.bottomhalf {
+    grid-row: 2/3;
+    grid-column: 1/2;
+    display: grid;
+    grid-template-columns: repeat(7, 45vw);
+    grid-template-rows: 1fr;
+    line-height: 2vw;
+    padding-left: 2vw;
+    font-family: Arial, Helvetica, sans-serif;
+    line-height: 2.2vw;
 
-.body-linethirtyfive {
-    margin-left: 16vw;
-}
+    h2 {
+        font-size: 2.6vw;
+        font-weight: bold;
+    }
 
-.body-linethirtysix {
-    margin-left: 13.6vw;
-}
+    div,
+    h3,
+    p,
+    li {
+        font-size: 1.7vw;
+        font-weight: 200;
+    }
 
-.body-linethirtyseven {
-    margin-left: 10.8vw;
-}
+    ul {
+        list-style-type: none;
+    }
 
-.body-linethirtyeight {
-    margin-left: 8.4vw;
 }
 
-.body-linethirtynine {
-    margin-left: 6vw;
-}
+/*left and right margins for body (set in div tags)*/
+.line__ {
```

Comment:
I recommend dropping the `body` from your classname, as it's not particularly meaningful. Here, I've renamed things to be BEM elements and nested the classes.